src/core/hle/libraries/libsystemservice/system_service.h

```diff
@@ -8,8 +8,19 @@ class SymbolsResolver;
 
 namespace Core::Libraries::LibSystemService {
 
+struct SceSystemServiceStatus {
+    s32 eventNum;
+    bool isSystemUiOverlaid;
+    bool isInBackgroundExecution;
+    bool isCpuMode7CpuNormal;
+    bool isGameLiveStreamingOnAir;
+    bool isOutOfVrPlayArea;
+    u08 reserved[];
```

Comment:
Εδω το reserved σου ειναι 0 bytes σύμφωνα με το standard, δεν ξερω τι θες να κανεις reserve

Reply:
nai den ksero kai ego etsi to ixe to doc..